api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/audit/NewClaimEvent.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.dhsc.htbhf.claimant.service.audit;
+
+import lombok.Builder;
+import uk.gov.dhsc.htbhf.claimant.model.ClaimStatus;
+import uk.gov.dhsc.htbhf.eligibility.model.EligibilityStatus;
+import uk.gov.dhsc.htbhf.logging.event.Event;
+
+import java.time.LocalDateTime;
+import java.util.Map;
+import java.util.UUID;
+
+import static uk.gov.dhsc.htbhf.claimant.service.audit.ClaimEventMetadataKey.CLAIMANT_ID;
+import static uk.gov.dhsc.htbhf.claimant.service.audit.ClaimEventMetadataKey.CLAIM_STATUS;
+import static uk.gov.dhsc.htbhf.claimant.service.audit.ClaimEventMetadataKey.ELIGIBILITY_STATUS;
+
+public class NewClaimEvent extends Event {
+
+    @Builder
+    public NewClaimEvent(UUID claimantId, ClaimStatus claimStatus, EligibilityStatus eligibilityStatus) {
+        super(ClaimEventType.NEW_CLAIM,
+                LocalDateTime.now(),
```

Comment:
Being picky here but can the formatting be adjusted so the code lines up.

Reply:
done